src/eth1/wallet.ts

```diff
@@ -0,0 +1,32 @@
+import defaults from "./defaults";
+import {ContractTransaction, ethers, Wallet} from "ethers";
+import {Provider} from "ethers/providers";
+import {number64} from "../types";
+import {BigNumber} from "ethers/utils";
+
+export class Eth1Wallet {
+
+  private wallet: Wallet;
+
+  public constructor(privateKey: string, provider?: Provider) {
+    if(!provider) {
+      provider = ethers.getDefaultProvider()
+    }
+    this.wallet = new Wallet(privateKey, provider);
+  }
+
+  /**
+   * Will deposit 32 ETH to eth2.0 deposit contract.
+   * @param address address of deposit contract
+   * @param amount amount to wei to deposit on contract
+   */
+  public async createValidatorDeposit(address: string, amount: BigNumber): Promise<string> {
```

Comment:
I'm thinking we can omit the `address` param, since it will be the same address everytime.

For Mainnet or testnets:
Someone will have deployed it -- so the address will be fixed

For local or testing:
In order to do that, we can have it deployed from the same private key (yes im proposing to push a private key to github ... lol), that way it will always deploy from the same account with the same nonce, thus the contract address will be the same (Assuming eth was given to that account on genesis).

Reply:
Cool idea, but I've found it's easier to put address param optional in cli, so we can ovveride it in development without messing with private key and hoping it will be same address ( if contract changes address will change as well). I've updated readme with commands for starting beacon chain in development which should make everything clearer

Reply:
interesting, are commander options by default optional?

Reply:
ok its clearly really late, `options` and `optional` go hand in hand

Reply:
Yup, they are optional by default, but they accept default value. Unfortunately they don't have any validations so I guess we would have to implement it ourselves. It also misses some beforeExit callback so we can close all connections sensibly (I've opened issue on their repo for that). 

Out of curiosity, did you consider https://github.com/yargs/yargs for cli?

Reply:
Yeah i did, I'm a bit of a crazy person when it comes to dependencies and commander has 0 where as Yargs has 11.
https://npmcompare.com/compare/commander,minimist,nomnom,optimist,yargs

Reply:
Thats not crazy, especially in js world 😊

Reply:
are there benefits to using yargs?

Reply:
Not really, I guess. They have some validation though

Reply:
Looks like it handles more complex CLIs a little nicer, having more features in general.
If we think our CLI might get fairly complex, or want to support more nested sorts of cli options, it might be useful to switch over at some point.

Reply:
Yeah - lets keep it simple, avoid over engineering